src/setting/dto/setting.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { IsString, IsNotEmpty } from "class-validator"
+import { ApiProperty } from "@nestjs/swagger"
+
+export class ChangePasswordDto {
+  @ApiProperty()
+  @IsString()
+  oldPassword: string
+  @ApiProperty()
+  @IsString() 
+  newPassword: string
+}
+export class ChangePhoneNoDto {
+  @ApiProperty()
+  @IsString()
+  newPhone: string
+  @ApiProperty(({minLength: 6, maxLength: 6}))
```

Comment:
This will work, but i not sure that is there is a fixLength to use?

Reply:
Checked with the class-validator docs. Don't think so. If anyone has found a way pls let me know.